pagopa/io-app
# feat: [PE-550] Revamp CGN Categories/All merchants screen with new DS

## Short description
This PR refactors the CGN categories and all merchants screen with to apply the new DS

## List of changes proposed in this pull request
- Refactors `CgnMerchantsCategoriesSelectionScreen.tsx`
- Move code to `CgnMerchantCategoriesListScreen.tsx`
- Refactors `CgnMerchantsListScreen.tsx`
- Removed `CGN_ROUTES.DETAILS.MERCHANTS.LIST` route

## How to test
Navigate to CGN details than check the list of categories and tab of merchants.

> [!important]
> `Icons` and `TabItem` style will be updated by DS package in a release after merging pagopa/io-app-design-system#236

https://github.com/pagopa/io-app/assets/3959405/b03924e9-9cd1-4ff2-81db-5655bffd8ac4
